sfs/array.py

```diff
@@ -34,6 +34,7 @@ class SecondarySourceDistribution(namedtuple('SecondarySourceDistribution',
     __slots__ = ()
 
     def __repr__(self):
+        """Return representation string."""
```

Comment:
I personally find that this docstring is utterly superfluous.

Sadly, `pydocstyle` recommends it, see https://github.com/PyCQA/pydocstyle/issues/60.